opx/sai_vm_event_log.h

```diff
@@ -18,9 +18,16 @@
 #ifndef __SAI_VM_EVENT_LOG_H__
 #define __SAI_VM_EVENT_LOG_H__
 
+#ifdef __cplusplus
+extern "C"
+{
+#endif
 #include "event_log.h"
 #include "event_log_types.h"
 #include "sai_event_log.h"
+#ifdef __cplusplus
+}
+#endif
```

Comment:
Adding extern "C" directives around #include directives seems very wrong.  Each header file should have extern "C" directives around their *own* public function/variable declarations